packages/admin-panel/src/App.js

```diff
@@ -39,7 +39,11 @@ export const App = ({ user, isBESAdmin }) => {
               <Switch>
                 {route.tabs.map(tab => (
                   <Route key={`${route.to}-${tab.to}`} path={`${route.to}${tab.to}`} exact>
-                    <tab.component getHeaderEl={getHeaderEl} isBESAdmin={isBESAdmin} />
+                    <tab.component
+                      getHeaderEl={getHeaderEl}
+                      isBESAdmin={isBESAdmin}
+                      isVizBuilderUser={isVizBuilderUser}
```

Comment:
Likewise here, can we just allow whichever pages/routes that need to know check if the user is BESAdmin/VizVuilderUser just work that out from the user's access policy rather than us pass this in to every component?